test/asset/raw_datasets.jsonl

```diff
@@ -21,9 +21,9 @@
+{"dataset_name": "Multi30k", "split": "train", "NUM_LINES": 29000, "MD5": {"train": "20140d013d05dd9a72dfde46478663ba05737ce983f478f960c1123c6671be5e", "valid": "a7aa20e9ebd5ba5adce7909498b94410996040857154dab029851af3a866da8c", "test": "0681be16a532912288a91ddd573594fbdd57c0fbb81486eff7c55247e35326c2"}, "URL": {"train": "http://www.quest.dcs.shef.ac.uk/wmt16_files_mmt/training.tar.gz", "valid": "http://www.quest.dcs.shef.ac.uk/wmt16_files_mmt/validation.tar.gz", "test": "http://www.quest.dcs.shef.ac.uk/wmt16_files_mmt/mmt16_task1_test.tar.gz"}, "first_line": "51cf1f8f5fd11c9299e1502e3ee3743a"}
```

Comment:
nit: Why are the train, test and valid hashes repeated for each split?

Reply:
Yes, I have noticed this in some other datasets too..We need to adapt the testing code little differently for datasets with unique URL for all the splits v/s datasets with different URLs for each split.